Mathlib/Algebra/Order/Monovary.lean

```diff
@@ -232,21 +244,26 @@ lemma Antivary.pow_right₀ (hg : 0 ≤ g) (hfg : Antivary f g) (n : ℕ) : Anti
 end LinearOrderedSemiring
 
 section LinearOrderedSemifield
-variable [LinearOrderedSemifield α] [LinearOrderedSemifield β] {s : Set ι} {f f₁ f₂ : ι → α}
-  {g g₁ g₂ : ι → β}
+variable [Semifield α] [LinearOrder α] [IsStrictOrderedRing α]
+  [Semifield β] [LinearOrder β] [IsStrictOrderedRing β]
+  {s : Set ι} {f f₁ f₂ : ι → α} {g g₁ g₂ : ι → β}
```

Comment:
Similarly here there looks to be a case for simply not making the assumptions `[Semifield β] [IsStrictOrderedRing β]` at this point (although it might permute the ordering of typeclass assumptions if you do this :-( )